config/settings.py

```
@@ -736,21 +745,38 @@ def _postgres_available() -> bool:
             "OPTIONS": {"options": "-c timezone=UTC"},
             "TEST": {
-                "NAME": f"{os.environ.get('POSTGRES_DB', 'postgres')}_test",
+                "NAME": f"{_postgres_name}_test",
             },
         }
     }
 else:
+    if DB_LOCK_CONFIGURED:
+        import warnings
+
+        warnings.warn(
```

Ruff (0.15.0)

[warning] No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)